web-src/src/components/GenerateButton.js

```diff
@@ -50,6 +50,69 @@ export function GenerateButton() {
   const saveResults = useSaveResults();
   const { formatMessage } = useIntl();
 
+  const closeToast = () => {
```

Comment:
Using this function improves user experience a little bit. While I'm not familiar with ToastContainer details, it looks like there is a queue and depending on different conditions, like auto closing timeout, if a user is hovering on the displaying toast, etc., it might be some queue of toasts for displaying, and it might look weird.

Reply:
I'll add one more point: this function isn't necessarily essential, but you can experiment with using it or not. Depending on the conditions mentioned earlier, you may find it useful. I would say having it doesn't hurt)